src/db/postgres.py

```diff
@@ -410,7 +417,23 @@ def parse_plan(self):
 
```

Comment:
Do we want to break or fail here, instead of `print(f'Unrecognized node header format: {pat_plan_node_header}')`?

If this code passed we will fail anyway because of `node == null`